ethik/base_explainer.py

```diff
@@ -254,6 +254,10 @@ def _explain(
         #   ],
         #   ...
         # ]
+
+        # Standard scale the features
+        X_test = (X_test - X_test.mean()) / X_test.std()
```

Comment:
I can you can create a utilitary function because it's also used in `compute_ksis`.

Reply:
I agree, but it's slightly different cases because in `compute_ksis` we are operating on a vector and we want to reuse the mean and the std. In `_explain` we are operating on a dataframe...